libra-node/src/main_node.rs

```diff
@@ -225,21 +225,21 @@ pub fn setup_environment(node_config: &mut NodeConfig) -> LibraHandle {
         metric_server::start_server(public_metric_host, public_metrics_port, true)
     });
 
+    // for state sync to send requests to mempool
+    let (mempool_channel, state_sync_requests) = channel(1_024);
```

Comment:
why do we need 1_024 buffer here?

Reply:
nit: can we rename variables inside of this function to more verbose
e.g. state_sync_to_mempool_sender